spec/lib/bgs/vnp_veteran_spec.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe BGS::VnpVeteran do
+  let(:user_object) { FactoryBot.create(:evss_user, :loa3) }
+  let(:user_hash) do
+    {
+      participant_id: user_object.participant_id,
+      ssn: user_object.ssn,
+      first_name: user_object.first_name,
+      last_name: user_object.last_name,
+      external_key: user_object.common_name || user_object.email,
+      icn: user_object.icn
+    }
+  end
+  let(:all_flows_payload) { FactoryBot.build(:form_686c_674) }
+  let(:current_marriage_details) do
```

Comment:
`current_marriage_details` isn't used anywhere - what gives? Can we delete it?

Reply:
Nice 👌 Like, left over from test(s) that were necessary before refactor/PR feedback.